client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx

```diff
@@ -70,8 +75,11 @@ class MappedDomainType extends React.Component {
 				}
 			);
 		} else {
+			setupInstructionsMessage = translate(
+				'Follow these instructions to set up your domain mapping:'
+			);
 			primaryMessage = translate(
-				'Your domain mapping has not been set up. You need to update your name servers at the company where you purchased the domain to:',
+				'In order to connect your domain to WordPress.com, log into your account at your domain registrar and update the name servers of your domain to use the following values:',
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?

Reply:
This is correct, we should use a `comment` instead of a `context` here (see PCYsg-56e-p2 for more information).

Reply:
I changed the `context`s for `comments`s 👍. I read the documentation but am still a little confused about when to use which one; should contexts be used for smaller pieces of text which might be ambiguous when translated in other languages?

Reply:
I would generally use comments 95% of the time. Here is [a comment](https://developer.wordpress.com/2015/04/23/wordpress-developers-test-your-i18n-internationalization-knowledge/#comment-6072) from @akirk where he summarized the difference between the two:

> A context adds information about the string itself (for example “verb”) or the area where the string is found (for example “column caption”).
> 
> A comment like /* translators: */ (mind the spelling) is meant to provide help with translating the string, for example, as we showed, by explaining what the variables stand for. Think of it as a personal message to translators.